packages/components/src/components/InfoItem/InfoItem.tsx

```diff
@@ -44,6 +44,7 @@ const Container = styled.div<ContainerProps>`
 
 export type InfoItemProps = AllowedFrameProps &
     AllowedTextProps & {
+        allowTextWrap?: boolean;
```

Comment:
What about do it with `textProps` for `ellipsisLineCount`? It's the same principle like `frameProps`

Check it here: `packages/components/src/components/typography/utils.tsx`

Reply:
good idea

Reply:
Why 0? I think it's correct this way: `ellipsisLineCount ?? 1`

Reply:
As we discussed: you need to pass then to this compoennt `ellipsisLineCount={0}` to disable line breaking (the inner comopnent has check `ellipsisLineCount > 0` to enable text overflow and ellipsis.